processor/src/tasks/processKicadPCB/index.ts

```diff
@@ -56,47 +66,47 @@ async function processKicadPCB(
     // Only plot gerbers if there's no `gerbers` key in kitspace.yaml.
     let gerbers = []
     if (kitspaceYaml.gerbers == null) {
-      gerbers = await plotKicadGerbers(outputDir, kicadPcbFile)
+      gerbers = await plotKicadGerbers(kicadPcbFile, tmpDir)
     }
 
     await layoutPromise
 
     const relativeKicadPcbFile = path.relative(inputDir, kicadPcbFile)
     const inputFiles = { [relativeKicadPcbFile]: { type: 'kicad', side: null } }
+    await fs.rm(layoutSvgPath)
     return { inputFiles, gerbers }
   } catch (error) {
     for (const file of filePaths) {
       job.updateProgress({ status: 'failed', file, error, outputDir })
     }
+    await fs.rm(layoutSvgPath, { force: true })
```

Comment:
This should be in the `finally` block like the `processPCB`.

Reply:
Changed in 179f9599d9dbbc1